src/porepy/fracs/fracture_network_2d.py

```diff
@@ -283,7 +191,7 @@ def mesh(
                 grids.
             remove_small_fractures: ``default=False``
 
-                Whether to remove small fractures.
+                DEPRECATED.
```

Comment:
Why not just remove parameter?

Reply:
In case someone uses the functionality without us knowing it. The current solution should be more user friendly.